components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -317,7 +389,29 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         }
         if (!this.state.desktopIde) {
           phase = StartPhase.Running;
-          statusMessage = <p className="text-base text-gray-400">Opening Workspace …</p>;
+
+          if (this.props.dontAutostart) {
+            // hide the progress bar, as we're already running
+            phase = undefined;
+            title = 'Running';
+
+            // in case we dontAutostart the IDE we have to provide controls to do so
```

Comment:
why this logic is not applicable when we run with desktop IDE as well?

Reply:
Looking at the code below I think all controls are already in place: there are "Open in Browser" and "Stop Workspace" buttons, so both branches should be equivalent.
The `desktopIDE` case basically never had autostart behavior from it's inception, because you have the IDE choice.

@akosyakov I'd really like to merge today. If there's anything I can do to speed things up (re-testing, sync discussion), please let me know. Thanks for the thorough review! :pray: